### PR TITLE
[docs] Update for 0.6.0

### PR DESCRIPTION
**Pull request by geropl**
DO NOT MERGE!

### CODE REVIEW

**Comment by meysholdt** [comment]
This looks really good already :)

I made a PR with some language improvement: https://github.com/gitpod-io/gitpod/pull/2858

Skimming the docs it was not clear to me where I should get my initial `values.custom.yaml` from. Should I copy it form somewhere? Should I start with an empty file? 

I think the [Recommended Configuration](https://deploy-preview-940--gitpod-website.netlify.app/docs/self-hosted/latest/install/install-on-kubernetes/#recommended-configuration) section would be a good place to introduce the `values.custom.yaml`.

And some menu items are broken because .md file names changed. IIRC the menus are declared in this repo.

**Comment by geropl** [comment]
@meysholdt 

> Skimming the docs it was not clear to me where I should get my initial values.custom.yaml from. Should I copy it form somewhere? Should I start with an empty file?

If you follow the steps one-by-one you'll get to "Configure Ingress", where it says: `Create a file values.custom.yaml with the following content:`

> I think the Recommended Configuration section would be a good place to introduce the values.custom.yaml.

It not only contains recommended but also required config (hostname, see ingress).

> And some menu items are broken because .md file names changed. IIRC the menus are declared in this repo.

Which ones?

**Comment by geropl** [comment]
@meysholdt Another thing, which I deem rather important: The website generator eats the enumerations and creates unsorted lists out of them. That makes it _really_ hard to follow through multiple steps.

Also, the spacing is looks really odd. Actually, the GitHub-rendered MD looks much better than the HTML the generator creates... :confused:

**Comment by meysholdt** [comment]
> Which ones?
"Configure a Domain" and "Configure HTTPS Certificates"

**Comment by meysholdt** [comment]
> Create a file values.custom.yaml
👍 

> It not only contains recommended but also required config (hostname, see ingress).

true, until we get mygitpod.com working again.

**Comment by geropl** [comment]
> "Configure a Domain" and "Configure HTTPS Certificates"

@meysholdt Forgot to ping: fixed.

**Review by meysholdt — APPROVED** [review]
